demos/code/code.js

```diff
@@ -158,8 +158,10 @@ Code.loadBlocks = function(defaultXml) {
  */
 Code.changeLanguage = function() {
   // Store the blocks for the duration of the reload.
+  // This should be skipped for the index page, which has no blocks and does
+  // not load Blockly.
```

Comment:
It is not clear to me what you are trying to say here. Which index page are you referring to?

Reply:
Actually, I did not change here and do not know why it is shown in my pull request.